data/sql_updates/alter_read_only_web_role.sql

```diff
@@ -3,8 +3,8 @@ $body$
 BEGIN
    IF NOT EXISTS (
       SELECT *
-      FROM   pg_catalog.pg_user
-      WHERE  usename = 'webro') THEN
+      FROM   pg_roles
+      WHERE  rolname = 'webro') THEN
```

Comment:
@arowla I had to make this change to get `alter_read_only_web_role` to run locally, but I haven't figured out why it wasn't working for me before. Does this change look all right to you?